.github/workflows/maas-api-ci.yml

```diff
@@ -8,6 +8,10 @@ on:
     paths:
       - 'maas-api/**'
 
+permissions:
```

Comment:
maybe, we should change these in the settings? To work, by default, with read-only permissions?
So that we avoid having this explicitly set here.

Reply:
That will be perfect.. currently the default permissions look like - 
```
GITHUB_TOKEN Permissions
  Actions: write
  Attestations: write
  Checks: write
  Contents: write
  Deployments: write
  Discussions: write
  Issues: write
  Metadata: read
  Models: read
  Packages: write
  Pages: write
  PullRequests: write
  RepositoryProjects: write
  SecurityEvents: write
  Statuses: write
```
With adding just `read` permissions (like in PR), we get the below and workflow runs fine -
```
 GITHUB_TOKEN Permissions
  Actions: read
  Contents: read
  Metadata: read
```
If we can set read-only explicitly in the settings that would be great.